pegasus/sites.v3/code.org/public/educate/resources/videos.md

```diff
@@ -82,6 +91,105 @@ We are building a growing library of educational videos available for re-use by
 
 <div style="clear: both;"></div>
 
+## Begin programming with JavaScript to make apps
+
+<div style="float:left; padding:10px; width:49%">
```

Comment:
longer term this might be easier to author as .haml with an array of video properties